crml/governance/src/lib.rs

```diff
@@ -286,4 +322,23 @@ impl<T: Config> Module<T> {
 	pub fn get_proposal_votes() -> Vec<(ProposalId, ProposalVoteInfo)> {
 		ProposalVotes::iter().collect()
 	}
+
+	pub fn check_council_account_validity(account: &T::AccountId) -> DispatchResult {
+		// Check the amount they have staked
+		let staked_amount: Balance = T::StakingAmount::active_balance(account.clone());
+		frame_support::log::info!("The amount staked by this account is: {:?}", staked_amount);
+
+		ensure!(
+			staked_amount >= Self::minimum_council_stake(),
+			Error::<T>::NotEnoughStaked
+		);
+
+		// Check their verified identities
+		let registration: u32 = T::Registration::registered_accounts(account.clone());
```

Comment:
account by ref here too

Reply:
nit: maybe rename `registered_accounts` to `registered_identity_count` so the term 'account' doesn't get confused here